dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -263,6 +264,26 @@ public Response aggregatedMetricsPut(@ApiParam(hidden = true) @Parameter(hidden
         return delegate.setAggregatedMetrics(id, versionId, platform, aggregatedMetrics);
     }
 
+    @GET
+    @UnitOfWork(readOnly = true)
+    @Path("/{id}/versions/{version_id}/aggregatedMetrics")
+    @Produces({MediaType.APPLICATION_JSON})
+    @ApiOperation(value = AggregatedMetricsGet.SUMMARY, notes = AggregatedMetricsGet.DESCRIPTION, authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Map.class)
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.SC_OK, message = AggregatedMetricsGet.OK_RESPONSE, response = Map.class),
+        @ApiResponse(code = HttpStatus.SC_NOT_FOUND, message = AggregatedMetricsGet.NOT_FOUND_RESPONSE, response = Error.class),
+        @ApiResponse(code = HttpStatus.SC_UNAUTHORIZED, message = AggregatedMetricsGet.UNAUTHORIZED_RESPONSE, response = Error.class)})
+    @Operation(operationId = "aggregatedMetricsGet", summary = AggregatedMetricsGet.SUMMARY, description = AggregatedMetricsGet.DESCRIPTION)
```

Comment:
Should add the openapi `ApiResponse` annotations like you have for swagger otherwise it doesn't show up in openapi.yaml

Reply:
It looks like the openapi `ApiResponse` annotations are the cause of the endpoint returning a String instead of a Map. I created a followup ticket to address this issue as it is affecting other endpoints as well. https://ucsc-cgl.atlassian.net/browse/SEAB-5373